core/util/path.ex

```diff
@@ -54,6 +54,10 @@ defmodule AntikytheraCore.Path do
     Path.join(gear_log_dir(gear_name), "#{gear_name}.log.gz")
   end
 
+  defun core_log_file_path(name :: v[String.t]) :: String.t do
```

Comment:
Like `gear_log_file_path/1`:
```suggestion
  defun core_log_file_path(name :: v[String.t]) :: Path.t do
```